src/Microsoft.AspNetCore.Mvc.Razor/TagHelpers/ITagHelperComponentContextInitializer.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+
+namespace Microsoft.AspNetCore.Mvc.Razor.TagHelpers
+{
+    /// <summary>
+    /// Provides methods to initialize <see cref="ViewContext"/> for <see cref="ITagHelperComponent"/>s.
```

Comment:
Given my recommendations in the implementation the docs would change a bit to be less about only setting view context.